static/app/views/performance/transactionSummary/transactionTags/types.tsx

```diff
@@ -0,0 +1,4 @@
+export enum XAxisOption {
+  DURATION = 'transaction.duration',
+  LCP = 'measurements.lcp',
+}
```

Comment:
Opted for an `enum` here to prevent typos in field names. I saw this pattern in a few places, so it seemed reasonable enough?